index.d.ts

```diff
@@ -737,6 +728,8 @@ export function execa<EncodingType extends EncodingOption = DefaultEncodingOptio
 /**
 Same as `execa()` but synchronous.
 
+Cannot use the following options: `all`, `cleanup`, `buffer`, `detached`, `serialization` and `signal`. Also, the `stdin`, `stdout`, `stderr`, `stdio` and `input` options cannot be a stream nor an iterable.
```

Comment:
The limitations with the `cleanup`, `buffer`, `detached`, `serialization` and `signal` was not previously documented.